src/components/Auth.test.js

```diff
@@ -0,0 +1,59 @@
+import {render, screen, waitFor} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import Auth from './Auth';
+
+jest.spyOn(window, 'fetch');
+
+describe('Auth', ()=>{
+    test('check if the success info is displayed when pass proper data', async ()=>{
+        render(<Auth />)
+
+        const login = screen.getByLabelText(/login:/i);
+        userEvent.type(login, 'jan@domena.pl');
```

Comment:
Ponieważ wielokrotnie używasz tego loginu to warto go zapisać do zmiennej i w odpowiednich miejscach użyć - wtedy jest mniejsze prawdopodobieństwo, że popełnisz błąd ;)

Reply:
zmienione